### PR TITLE
Improve support for handling decorators in python

### PR DESCRIPTION
**Pull request by flenter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,25 +22,26 @@ This extension contributes the following settings:
 
 ## Known Issues
 
-Right now, the detection for the `@autometrics` decorator is fairly simplistic. This means that the detection will sometimes fail if you use multiple decorators on a single function (the autometrics decorator should be right above the function definition).
+If you mix tabs and spaces in your document, the detection of the autometrics might fall as it is assumed that the whole document is indented the same way
```
Comment (emschwartz) [inline README.md:25]: That seems like a strange limitation. Can't you just look for both?

Comment (flenter) [reply README.md:25]: The issue with mixing tabs and spaces is that depending on the platform a tab equals either 4 or 8 spaces. In general I would say that mixing them is bad practice.

Comment (emschwartz) [reply README.md:25]: Sure. But "be strict in what you produce but liberal in what you accept"

Comment (flenter) [reply README.md]: I've added code that tries to guess what the indentation you're aiming for

**Review by emschwartz — APPROVED** [review]

**Pull request merged**